models/articles.model.js

```diff
@@ -0,0 +1,12 @@
+const format = require("pg-format");
+const db = require("../db/connection");
+
+exports.selectArticleById = (req) => {
+  const { article_id } = req.params;
```

Comment:
The req.params should be destructured in the controller. The controller takes care of the req and res, so it belongs in there 👍